Examples/Image/Classification/AlexNet/readlabels.py

```diff
@@ -0,0 +1,85 @@
+import sys
```

Comment:
This name does not do justice to what this scrip is doing. It creates all the necessary files. Maybe change the name to reflect that. Also use title case for filename ('_' are OK to use).